common/web/gesture-recognizer/src/test/auto/headless/segmentConstruction.js

```diff
@@ -527,7 +527,7 @@ describe("Iterative Segment Construction (ConstructingSegment)", function() {
         assert.isTrue(constructor.isCompatible(followup)); // To verify the following method's precondition.
         assert.isTrue(constructor.updatePendingSubsegment(followup));
 
-        followup = turtle.move(90, 100, 20, 20);
+        followup = turtle.move(90, 1, 20, 20);
```

Comment:
The unit test's title indicated that this should have been a `1`, not a `100`:

```
 hold -> 2px move -> hold -> 1px move
```

The changed value reflects the number of pixels the simulated touch-move should travel.

Reply:
Ha ha, was wondering about this, but figured you knew what you were doing ;-)